package.json

```diff
@@ -32,9 +32,9 @@
     "sinon-chai": "^2.14.0"
   },
   "devDependencies": {
-    "webpack": "3"
+    "webpack": "3 || 4"
```

Comment:
this should be a peer dependency...

Reply:
did you check the webpack changelog to see if there are any relevant breaking changes? or try running this on webpack 4? I scanned it and didn't see anything, but I might have missed something, because the changelog is really long

Reply:
actually i'm going to change this commit to only upgrade karma. there are more changes i'll need to make for webpack and i'll be able to link it to a working build